packages/strapi-design-system/playwright.config.js

```diff
@@ -0,0 +1,40 @@
+const { devices } = require('@playwright/test');
+
+const config = {
+  testMatch: '**/__tests__/*.e2e.js',
+  timeout: 10000,
+  // Forbid test.only on CI
```

Comment:
Why not always forbidding it?

Reply:
🤔  I don't understand your comment here. Do you mean forbid `.only` on local too?
On local, if you add a `.only` on a test, I guess you want to run only that test 😅 . 
On the CI, you want to run the whole test suite in all cases. (forbidOnly will trigger an error if `.only`)

Reply:
I mean, if we always forbid the `.only`, you'll not be able to run only one test if you want (on local).

Reply:
Sure but I guess that you can run your test locally and then the precommit hook can fail if your test contains `.only`